packages/plugins/docs-ui/src/lib/components/tree/docs-tree.component.html

```
@@ -24,8 +24,29 @@
+		     sidebar's way back to "everything" (`00-product-spec.md` §6.9 R-TRE-01). A real
+		     <button> so focus, Enter and Space come from the platform; the SCSS only undoes
+		     the user-agent chrome, so it still renders as the section label it always was. -->
+		<button type="button" class="docs-tree-section-title docs-tree-root" (click)="goToRoot()">
+			{{ 'DOCS.TREE.ALL_DOCUMENTS' | translate }}
+		</button>
+		<div class="docs-tree-empty" *ngIf="!nodes.length">
+			<span>{{ 'DOCS.TREE.EMPTY' | translate }}</span>
+			<!-- An empty tree has no node to raise a context menu on, so without these the
+			     first folder/page of a brand-new organization is unreachable from the sidebar. -->
+			<div class="docs-tree-empty-actions" *ngIf="canCreate">
+				<button nbButton ghost size="tiny" (click)="createAtRoot(kindEnum.FOLDER)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)